### PR TITLE
Tag Cloud: Remove `strtolower` around taxonomy name

### PR DESCRIPTION
**Pull request by Soean**
## Description
Bildschirmfoto 2019-06-12 um 11 57 32

In German we have `Kategorien` (categories) and `Schlagwörter` (tags). The first letter needs to be uppercase.

### CODE REVIEW

**Comment by Soean** [comment]
I added the copy review label:
Is it okay to use the uppercase `Tags` instead of `tags`?
`Your sites doesn’t have any Tags, so there’s nothing to display here at the moment.`

**Comment by michelleweber** [comment]
Hmm. That doesn't really work in English; "tags" isn't a word that would be capitalized except in a headline/title.

**Comment by Soean** [comment]
Thanks for your feedback. So we have to change the sentence.

**Comment by aduth** [comment]
> Thanks for your feedback. So we have to change the sentence.

Are you aware of any prior art for how these issues have been dealt with in the past, as a general point of localization best practices? I'd wonder if in the worst case it might be reasonable to have logic to switch on the locale in deciding the appropriate treatment for the word.

**Comment by ocean90** [comment]
Have you thought about using a more generic string instead? 

> Your site doesn't have any items in the selected taxonomy, so there's nothing to display here at the moment.

Otherwise see [`get_post_type_labels()`](https://developer.wordpress.org/reference/functions/get_post_type_labels/) and [`get_taxonomy_labels()`](https://developer.wordpress.org/reference/functions/get_taxonomy_labels/). Both can be used to define new labels which depend on the post type/taxonomy name.

**Comment by melchoyce** [comment]
I'd like to avoid using words like "taxonomy" if we can, since it's a pretty technical word.

**Comment by ocean90** [comment]
But it's something the user has selected in the block settings, see https://github.com/WordPress/gutenberg/blob/39f568faf2e62c57736bb80d7088a996eb067944/packages/block-library/src/tag-cloud/edit.js#L69-L74.

**Comment by melchoyce** [comment]
Yeah, I mean tbh I'd love to find a better way to write that, too.

**Comment by melchoyce** [comment]
@michelleweber How do you feel about @ocean90's suggestion? I might just be overthinking this.

**Comment by ocean90** [comment]
Not sure about the label of the select field but here are some other ideas:

> With the current settings there’s nothing to display here at the moment.

> There’s no content that matches your selection, yet.

**Comment by michelleweber** [comment]
I'd definitely like to avoid using the word "taxonomy."  We could definitely go general with something like "There's no content to show here yet," but if there's a way to include specific messaging that both explains and nudges the user toward a next step/way to resolve, that would be far, far preferable.

**Comment by Soean** [comment]
Thanks for the Feedback, I updated the text to: `There's no content to show here yet.`. This is translatable in all languages and doesn't include the word 'taxonomy'.

**Comment by michelleweber** [comment]
"No content to show yet" is fine, but it would be really great if, as @aduth suggests, we figure out ways to fine-tune localization. Ideally we're describing things as clearly as possible, and in this case, that means using the word "tags."

**Comment by swissspidy** [comment]
Well the only way to do that is by extending the list of available taxonomy labels, as mentioned earlier in this thread. See `get_taxonomy_labels()` in core.

So just like we have labels like `Add New Tag` and `Add New Category`, there would need to be new labels `Your site doesn't have any tags, so there’s nothing to display here at the moment.` / `Your site doesn't have any categories, so there’s nothing to display here at the moment.`

I'm not sure if it's really worth it to add such a new label for this specific use area.

File: packages/block-library/src/tag-cloud/index.php [hunk 1]
```diff
@@ -31,13 +31,7 @@ function render_block_core_tag_cloud( $attributes ) {
...
 	if ( ! $tag_cloud ) {
 		$labels    = get_taxonomy_labels( get_taxonomy( $attributes['taxonomy'] ) );
-		$tag_cloud = esc_html(
-			sprintf(
-				/* translators: %s: taxonomy name */
-				__( 'Your site doesn&#8217;t have any %s, so there&#8217;s nothing to display here at the moment.' ),
-				strtolower( $labels->name )
-			)
-		);
+		$tag_cloud = esc_html( __( 'There\'s no content to show here yet.' ) );
```
Comment (ocean90) [inline packages/block-library/src/tag-cloud/index.php:34]:
```suggestion
		$tag_cloud = __( 'There&#8217;s no content to show here yet.' );
```

**Review by ocean90 — DISMISSED** [review]

**Comment by Mamaduka** [comment]
Hi, @Soean

Can you rebase the branch? I think we can merge this after https://github.com/WordPress/gutenberg/pull/16112#discussion_r424036007 is addressed.

**Comment by Soean** [comment]
@Mamaduka thanks, I rebased the branch.

**Review by Mamaduka — APPROVED** [review]
Thanks for the follow-up, @Soean 🙇

**Pull request merged**